### PR TITLE
amend signMember endpoint to just take memberId rather than memberId …

### PR DESCRIPTION
**Pull request by AlexSJ1990**
…and ssn

### CODE REVIEW

File: member-service/src/main/java/com/hedvig/memberservice/services/SigningService.java [hunk 1]
```diff
@@ -128,23 +126,30 @@ public MemberSignResponse startWebSign(final long memberId, final WebsignRequest
...
+  public MemberSignUnderwriterQuoteResponse signUnderwriterQuote(final long memberId) {
 
-    Optional<SignedMemberEntity> existing = signedMemberRepository.findBySsn(request.getSsn());
+    Optional<MemberEntity> member = memberRepository.findById(memberId);
 
-    if (existing.isPresent()) {
-      throw new MemberHasExistingInsuranceException();
+    String ssn = null;
+
+    if (member.isPresent()) {
```
Comment (meletios) [inline member-service/src/main/java/com/hedvig/memberservice/services/SigningService.java]: Shall we invert this if statement? and throw if the member doesnt exist?

Comment (AlexSJ1990) [reply member-service/src/main/java/com/hedvig/memberservice/services/SigningService.java]: yes good point

**Comment by johantjellden** [comment]
@AlexSJ1990 Should we merge this?

**Comment by johantjellden** [comment]
Closing this now.

**Pull request closed without merging**